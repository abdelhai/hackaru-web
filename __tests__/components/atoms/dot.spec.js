import Factory from '@/__tests__/__setups__/factory';
import Dot from '@/components/atoms/dot';

describe('Dot', () => {
  it('render correctly', () => {
    const wrapper = new Factory(Dot, {
      propsData: {
        color: '#ff0000'
      }
    }).shallow();
    expect(wrapper.element).toMatchSnapshot();
  });
});
