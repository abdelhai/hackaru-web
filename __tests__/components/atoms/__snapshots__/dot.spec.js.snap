// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot render correctly 1`] = `
<span
  class="dot"
  style="background-color: rgb(255, 0, 0);"
/>
`;
